From 788ff67f8463288673019bd55d5302e1c55c38a5 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:17:22 +0800
Subject: [PATCH] Test case1 for Change package guid under Define section

Description: Change package guid under Define section (to "BA1C6956-2CEF-423a-BF4D-B5025A894C88") in MdeModulePkg/MdeModulePkg.dec.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/MdeModulePkg.dec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/MdeModulePkg.dec b/MdeModulePkg/MdeModulePkg.dec
index 843e963ad3..8743e245db 100644
--- a/MdeModulePkg/MdeModulePkg.dec
+++ b/MdeModulePkg/MdeModulePkg.dec
@@ -18,7 +18,7 @@
   DEC_SPECIFICATION              = 0x00010005
   PACKAGE_NAME                   = MdeModulePkg
   PACKAGE_UNI_FILE               = MdeModulePkg.uni
-  PACKAGE_GUID                   = BA0D78D6-2CAF-414b-BD4D-B6762A894288
+  PACKAGE_GUID                   = BA1C6956-2CEF-423a-BF4D-B5025A894C88
   PACKAGE_VERSION                = 0.98
 
 [Includes]
-- 
2.27.0.windows.1

